beacon/blockchain/mocks/storage_backend.mock.go

```diff
@@ -1,12 +1,12 @@
-// Code generated by mockery v2.49.0. DO NOT EDIT.
```

Comment:
moved here from backend mocks, since this is the only place where this mock is used (and the only used mock among the backend ones)